docs/guides/apache-sites-enabled.md

```diff
@@ -79,11 +78,11 @@ The configuration file configuration content would look something like this:
 ```
 Once the file is created, we need to write (save) it with: `shift : wq`
 
-In our example above, the wiki site is loaded from the html sub-directory of _com.ourownwiki.www_, which means that the path we created in _/var/www_ (above) will need some additional directories to satisfy this:
+In our example above, the wiki site loads from the HTML sub-directory of _com.ourownwiki.www_, which means that the path we created in _/var/www_ (above) will need some additional directories to satisfy this:
 
 `mkdir -p /var/www/sub-domains/com.ourownwiki.www/html`
 
-... which will create the entire path with a single command. Next we would want to install our files to this directory that will actually run the web site. This could be something you made yourself, or an installable web application (in this case a wiki that you downloaded). 
+These directories will create the entire path with a single command. Then we would want to install our files to this directory which will run the website. It could be something you made yourself or an installable web application (in this case, a wiki that you downloaded).
 
 Copy your files to the path above:
```

Comment:
"These directories" is not correct. That's a command, not a directory. The command creates the entire path at once.

Reply:
This was another error which I could not correct after I pushed the file, but the grammar in the first paragraph is not good. A sentence never starts with an ellipsis and the comma after ‘yourself ‘were issues. And that is why I altered the sentence. Also once again the passive sense is being used and it should not.

Reply:
n closing, I have ‘no horses’ in this race and it does not matter to me if changes are accepted or not. However, I spent about four hours of my time on this, correcting the grammar and I even posted on the forum about the two errors and that I did not know how to correct them once the file was pushed. The ‘website web site’ thing is nonsense, because Americans have so warped the language that almost no one cares any more about speaking or writing in the most understandable way possible. Web site was not really right when it was first used and now it is almost never used, in fact most of the style manuals use website or Website. I used CENTOS when it first came out and then got in to other things. Just when I started using it again, this thing happened with Redhat. I decided to stay with Rocky and I just want to see the best documentation possible and while I do not claim what I wrote is the best, it is significantly better that what was there and someone will probably come along and make mine better again.
Some of this would not have happened if I understood Github a little better, but I was waiting for it to be pushed through and then correct those two errors. Or if there is a way to edit it while it is where it is, then I can fix I; With the exception of the two errors, I challenge anyone to show me where or how the meaning is changed in the corrected versions as compared to the originals.

Reply:
I actually posted on the forum to you about the push of he same PR. but you might not have seen it I guess.  I have a local copy that I use to build PR's against. Before I build a PR, I create a separate branch that is a copy of main. That way, I'm leaving main alone. When I push it, I push it to my fork of the documentation repository, which will "see" that this is different than the main and prompt me to create a PR. From the recent history on my machine, for instance, I created a branch called "rl_change_top_level" (it can be named whatever you want). I then pushed it to my fork of the documentation repository:

git push origin rl_change_top_level 

(my fork is origin and the rocky documentation is remote is named "upstream")

I then get this back from git:

remote: Create a pull request for 'rl_change_top_level' on GitHub by visiting:
remote:      https://github.com/sspencerwire/documentation/pull/new/rl_change_top_level

I copy and paste that URL into my browser and it gives me the Pull Request dialog so that I can create the Pull Request against Rocky's main.

Now let's say that I notice that I made a mistake and want to fix it. My PR is now Open, but hasn't been merged yet. Simply make the changes on your local copy of the documentation within the same branch (in this case rl_change_top_level) and then commit the changes and push the same branch to your fork of the repository again. This will update the existing PR. Lots of steps to remember. It takes awhile to get up-to-speed with git. I'm not there yet by any stretch.

Reply:
I didn't mean to ruffle your feathers, nor do I mean to continue to do so, but I stand firm that the basic issues that I pointed out change the meaning of the document.